scripts/kmodtool

```diff
@@ -333,7 +333,7 @@ print_customrpmtemplate ()
 {
 	for kernel in ${1}
 	do
-		if [[ -e "${buildroot}/usr/src/kernels/${kernel}" ]] ; then
+		if [[ -e "${buildroot}/usr/src/kernels/${kernel}" ]]  && [[ "${kernel}" =~ el|fc ]]; then
```

Comment:
Unfortunately this is going to break some RedHat derivatives.  Could we instead invert the conditional and check for a user-build-kernel and then a Fedora/RH kernel.

Reply:
Inverting the conditional seems like it could work. I'll test it out and update this PR when I get the chance. Thanks!